web/packages/core/src/load-ruffle.ts

```diff
@@ -27,13 +32,51 @@ async function fetchRuffle(config: Config): Promise<typeof Ruffle> {
     // libraries, if needed.
     setPolyfillsOnLoad();
 
+    const extensionsSupported: boolean = (
+        await Promise.all([
+            bulkMemory(),
+            simd(),
+            saturatedFloatToInt(),
+            signExtensions(),
+        ])
+    ).every(Boolean);
+
+    if (extensionsSupported) {
+        console.log(
+            "All necessary WASM extensions are available, loading the fancy WASM module"
```

Comment:
maybe let's not log the happy case, let's not logspam random websites too much :)

Reply:
Alright.